services/terrain/project.clj

```diff
@@ -7,13 +7,13 @@
       (string/trim (:out (sh "git" "rev-parse" "HEAD")))
       ""))
 
-(defproject org.iplantc/donkey "5.0.0-SNAPSHOT"
+(defproject org.iplantc/terrain "5.0.0-SNAPSHOT"
   :description "Framework for hosting DiscoveryEnvironment metadata services."
-  :url "https://github.com/iPlantCollaborativeOpenSource/Donkey"
+  :url "https://github.com/iPlantCollaborativeOpenSource/Terrain"
```

Comment:
I forgot about this project setting too. I think you're right and they should all now point to `DE`.